src/examples/work-with-cache-pipenv.yml

```diff
@@ -0,0 +1,24 @@
+description: |
+  An example of working with the Pipenv cache on CircleCI to speed up builds.
+usage:
+  version: 2.1
+  orbs:
+    python: circleci/python@x.y.z
```

Comment:
@KyleTryon @gmemstr I think we just decided to use real versions here?

Reply:
Yup. I'll change that, but I was just thinking. How do I know what the next version is? Could we parameritize/auto-populate a field? Not sure what makes sense in my ignorance.